src/views/record-browser/record-browser.html

```diff
@@ -8,12 +8,12 @@
 	<div class="help-text" ng-hide="recordBrowser.results">
 		<h1>Use the Query Builder to the left to retrieve your data.</h1>
 	</div>
-
```

Comment:
Unnecessary change. If this is intentional, it should be its own commit.

For example:
`style(record-browser): remove unnecessary blank line`